radeon-profile/dxorg.cpp

```diff
@@ -555,7 +555,7 @@ GPUPwmStruct dXorg::getPwmSpeed() {
 
     QFile f(hwmonAttributes.pwm1);
 
-    if (f.open(QIODevice::ReadOnly)) {
+    if (f.exists() && f.open(QIODevice::ReadOnly)) {
```

Comment:
When f.open() fails, it throws a warning on standard out (in this case `QFSFileEngine::open: No file name specified`), and since this code is executed on evry update this causes [a very verbose and annoying list of warnings](https://user-images.githubusercontent.com/8406735/27236221-6384d6c8-52c4-11e7-9d84-20b4d1429bb8.png) .